Dockerfile

```diff
@@ -1,26 +1,28 @@
 ARG GOLANG_VERSION=1.22.0
 
-ARG GOOS=linux
-ARG GOARCH=amd64
+# TARGETOS and TARGETARCH will be populated at 'docker build'
+ARG TARGETOS  
```

Comment:
Why rename these?

Reply:
Docker Buildx sets the `TARGETOS` and `TARGETARCH` to correct values. It felt like a better reference

Reply:
Ok. Thanks for the explanation.